project/src/main/puzzle/Playfield.tscn

```diff
@@ -182,7 +182,7 @@ script = ExtResource( 5 )
 puzzle_tile_map_path = NodePath("../TileMap")
 
 [node name="ShadowTexture" type="TextureRect" parent="."]
-modulate = Color( 1, 1, 1, 0.196078 )
+modulate = Color( 1, 1, 1, 0.313726 )
```

Comment:
Document why this value was changed.

Reply:
Added commit message, `Made piece shadows slightly darker so that ghost piece would be more
visible.`